opus/application/apps/search/models.py

```diff
@@ -1454,6 +1454,7 @@ class ObsInstrumentCoiss(models.Model):
     target_desc = models.CharField(max_length=75)
     combined_filter = models.CharField(max_length=30)
     camera = models.CharField(max_length=1)
+    duration = models.FloatField(blank=True, null=True)
```

Comment:
There should be no changes to models.py (either here or in ObsOccultation below). We aren't actually adding new fields to the database.